samples/client/petstore/typescript-node/default/.openapi-generator/FILES

```diff
@@ -1,14 +1,12 @@
 .gitignore
 api.ts
+api/advancedApi.ts
```

Comment:
has this been generated on Windows by any chance? If yes, another reference to #19726

Reply:
Indeed not using platform-independent path. Will this be fixed by #19726?

Reply:
^ @wing328 #19726 is waiting for you 😬 

> Indeed not using platform-independent path. Will this be fixed by #19726?

yes, hopefully.